CHANGELOG.md

```diff
@@ -8,9 +8,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased] - ReleaseDate
 ### Added
```

Comment:
Does the release process in this repo automatically add the version header here and in the links in the bottom?

Reply:
I add them when I do the `cargo release` from main, which updates the CHANGELOG, tags, and pushes everything and then lets CI build the release artifacts.

Reply:
ah ok!